src/components/nve-switch/nve-switch.component.ts

```diff
@@ -0,0 +1,148 @@
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import { CSSResultArray, html, LitElement, TemplateResult } from 'lit';
+import { watch } from '../../utils/watch';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { live } from 'lit/directives/live.js';
+import { classMap } from 'lit/directives/class-map.js';
+import styles from './nve-switch.styles';
+/**
+ * En toggle-button med støtte for ikoner.
+ *
+ * @slot label - Tekst etter switch
+ * @slot onicon - Det første ikonet (til venstre). Synlig når status er PÅ
+ * @slot officon - Det andre ikonet (til høyre). Synlig når status er AV
+ *
+ * @event blur
+ * @event change
+ * @event input
+ * @event focus
+ */
+@customElement('nve-switch')
+export default class NveSwitch extends LitElement implements INveComponent {
+  constructor() {
+    super();
+  }
+  @property({ reflect: true, type: String }) testId: string = '';
+
+  /** Hidden checkbox som holder state */
+  @query('input[type="checkbox"]') input!: HTMLInputElement;
+  @state() private hasFocus = false;
+  @property() title = ''; // make reactive to pass through
+
+  /** Navn på switch */
+  @property() name = '';
+
+  /** Switchens verdi, bruk i forms */
```

Comment:
Hvorfor trenger du dette? Hvordan bruker man dette i forms? 
Blir den "true" eller "false" eller noe?
Kan man sette verdien med denne også?

Reply:
Dette er grunnleggende HTML, alle form-elementer har name / value som brukes når man submitter.

For en checkbox er name det samme på alle i en gruppe, value er ulikt og browser bruker checked for å si at de skal submittes (value er fast i en form dersom du ikke gjør noe spesielt, det er også name).

For et sett med checkboxer i en form:

<form>
  <input type="checkbox" name="topping" value="pepperoni" />
  <input type="checkbox" name="topping" value="peppers" />
  <input type="checkbox" name="topping" value="pineapple" disabled />
</form>

Tilsvarende: 
<form>
  <nve-switch name="topping" value="pepperoni" />
  <nve-switch name="topping" value="peppers" />
  <nve-switch name="topping" value="pineapple" disabled />
</form>

Reply:
Takk! Beklager min uvitenhet:)